src/server/components/SeoMetaTags.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {buildUrl, getNodeProps, useServerContext} from '@jahia/js-server-core';
+
+export const SeoMetaTags = () => {
+    const {currentNode, currentResource, renderContext} = useServerContext();
+
+    const isDisplayableNodeType = currentNode.isNodeType('jnt:page') ||
+        currentNode.isNodeType('jmix:mainResource');
+    if (!isDisplayableNodeType) {
+        return;
+    }
+
+    const {
+        'jcr:title': seoTitle,
+        'jcr:description': seoDescription,
+        openGraphImage,
+        seoKeywords
+    } = getNodeProps(currentNode, [
+        'jcr:title',
+        'jcr:description',
+        'openGraphImage',
+        'seoKeywords'
+    ]) || {};
+    const locale = currentResource.getLocale().getLanguage();
+    const absOgImageUrl = openGraphImage?.getAbsoluteUrl(renderContext.getRequest());
+
+    const getAbsoluteUrl = node => {
+        const server = renderContext.getURLGenerator().getServer();
+        const relUrl = buildUrl({path: node.getPath()}, renderContext, currentResource);
+        return `${server}${relUrl}`;
+    };
+
+    return (
+        <>
+            {absOgImageUrl && <meta property="og:image" content={absOgImageUrl}/>}
+            {seoTitle && <meta property="og:title" content={seoTitle}/>}
+            <meta property="og:url" content={getAbsoluteUrl(currentNode)}/>
+            {seoDescription &&
+                <>
+                    <meta property="og:description" content={seoDescription.toLocaleString(locale) || seoDescription}/>
+                    <meta name="description" content={seoDescription.toLocaleString(locale) || seoDescription}/>
+                </>}
+            {seoKeywords?.length && <meta name="keywords" content={seoKeywords.map(kw => kw.toLocaleString(locale)).join(',')}/>}
```

Comment:
Should we sanitize the keywords here in case they contain `,`?
Or maybe this is done upfront when storing the properties in the JCR?
Or maybe, it's OK to ignore this case? 🙂

Reply:
for me this is ok. it's already sanitized when adding the tags in content editor.